spec/support/authenticate.rb

```diff
@@ -0,0 +1,10 @@
+module Authenticate
+  def authenticate_user user
+    allow_any_instance_of(ApiController).to receive(:authenticate_api_user).and_return(true)
+    allow_any_instance_of(ApiController).to receive(:current_user).and_return(@user)
+  end
+end
+
+RSpec.configure do |config|
+  config.include Authenticate
```

Comment:
The other helper, `ResponseJSON`, also had the configuration in the helper file so I wanted to keep consistency.
But I agree, having configs in `/support/rails_helper.rb` is cleaner.